oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandlerTest.java

```diff
@@ -1585,6 +1574,18 @@ void shouldReturnErrorWhenInvalidPromptValuesArePassed(
                         pair("description", expectedError.getDescription()));
     }
 
+    @Test
+    void shouldHandleMissingTxmaHeader() throws JOSEException {
+        Map<String, String> requestParams = buildRequestParams(null);
+        var event = new APIGatewayProxyRequestEvent();
+        event.setHttpMethod("GET");
+        event.setQueryStringParameters(requestParams);
+        event.setRequestContext(
+                new ProxyRequestContext()
+                        .withIdentity(new RequestIdentity().withSourceIp("123.123.123.123")));
+        var response = makeHandlerRequest(event);
```

Comment:
Can we assert on something here?

Reply:
makeHandlerRequest does some assertions, this test has been moved as the logic has been moved to a helper class